manifests/state-doca-telemetry-service/0010-doca-telemetry-service.yaml

```diff
@@ -18,6 +18,9 @@ spec:
       # Required by the sysfs collector only.
       hostNetwork: true
       serviceAccount: doca-telemetry-service
+      nodeSelector:
+        feature.node.kubernetes.io/pci-15b3.present: "true"
+        network.nvidia.com/operator.mofed.wait: "false"
```

Comment:
Will the label exist if MOFED is not configured in NCP?

Reply:
operator always creates this label